miteD/service/service.py

```diff
@@ -76,8 +101,8 @@ def _get_handler(self, msg):
                     raise NotImplementedError(subject)
 
             async def _expose_api_version(self, api, api_version):
-                subject = '{}.{}.*'.format(api, api_version.replace('.', '_'))
-                self._logger.info('listening for messages on ' + subject)
+                subject = '{}.{}.{}.{}.*'.format('rpc', 'service', api, api_version)
```

Comment:
wouldn't `api_version` here need a `format_version_str(api_version)` instead?

Reply:
the _expose_api_version() method is called with 'api_version' obtained from self._versions member of the Service class: https://github.com/twoporeguys/miteD/pull/7/files/cd883bf98b5883f68e145139c7717412598c8f32#diff-4fb074af1caf860b0db2ed17358faa79R83

And values there get formatted here already:
https://github.com/twoporeguys/miteD/pull/7/files/cd883bf98b5883f68e145139c7717412598c8f32#diff-4fb074af1caf860b0db2ed17358faa79R54